ddl/job_table.go

```diff
@@ -562,11 +561,9 @@ func job2SchemaNames(job *model.Job) []string {
 		names = append(names, strings.ToLower(job.SchemaName))
 		names = append(names, oldSchemaName.O)
 		return names
-	case model.ActionRenameTables:
-		// TODO: Get this action's schema names.
-	case model.ActionExchangeTablePartition:
-		// TODO: Get this action's schema names.
 	}
+	// TODO: consider about model.ActionRenameTables and model.ActionExchangeTablePartition, which need to get the schema names.
```

Comment:
Better one.

If not necessary, we'd better not add the code. "// TODO:" is a option.

It would be much better to talk about such kind of potential possibility in the design procedure, as much as possible.

Reply:
Yeah, but here I think it needs to be added.